src/android/FirebaseMessagingPlugin.java

```diff
@@ -220,7 +220,13 @@ private void findChannel(String channelId, CallbackContext callbackContext) thro
             callbackContext.success(new JSONObject()
                 .put("id", channel.getId())
                 .put("name", channel.getName())
-                .put("description", channel.getDescription()));
+                .put("description", channel.getDescription())
+                .put("importance", channel.getImportance())
+                .put("badge", channel.canShowBadge())
+                .put("light", channel.shouldShowLights())
```

Comment:
```suggestion
                .put("showLights", channel.shouldShowLights())
```

Reply:
Same situation; the name `light` was derived from the part where you set the channel properties:

`src/android/FirebaseMessagingPlugin.java:179`
```java
channel.enableLights(options.optBoolean("light", false));
```

Shouldn't the names when *getting* the values be the same as when  *setting* the values?